bgfx.idl

```diff
@@ -321,7 +351,45 @@ struct.EncoderStats
 
 --- Renderer statistics data.
 struct.Stats
-	.cpuTimeFrame "int64_t" [[CPU time between two `bgfx::frame` calls.]]
+	.cpuTimeFrame            "int64_t"       [[CPU time between two `bgfx::frame` calls.]]
+	.cpuTimeBegin            "int64_t"       [[Render thread CPU submit begin time.]]
+	.cpuTimeEnd              "int64_t"       [[Render thread CPU submit end time.]]
+	.cpuTimerFreq            "int64_t"       [[CPU timer frequency. Timestamps-per-second]]
+	.gpuTimeBegin            "int64_t"       [[GPU frame begin time.]]
+	.gpuTimeEnd              "int64_t"       [[GPU frame end time.]]
+	.gpuTimerFreq            "int64_t"       [[GPU timer frequency.]]
+	.waitRender              "int64_t"       [[Time spent waiting for render backend thread to finish issuing draw commands to underlying graphics API.]]
+	.waitSubmit              "int64_t"       [[Time spent waiting for submit thread to advance to next frame.]]
+	.numDraw                 "uint32_t"      [[Number of draw calls submitted.]]
+	.numCompute              "uint32_t"      [[Number of compute calls submitted.]]
+	.numBlit                 "uint32_t"      [[Number of blit calls submitted.]]
+	.maxGpuLatency           "uint32_t"      [[GPU driver latency.]]
+	.numDynamicIndexBuffers  "uint16_t"      [[Number of used dynamic index buffers.]]
+	.numDynamicVertexBuffers "uint16_t"      [[Number of used dynamic vertex buffers.]]
+	.numFrameBuffers         "uint16_t"      [[Number of used frame buffers.]]
+	.numIndexBuffers         "uint16_t"      [[Number of used index buffers.]]
+	.numOcclusionQueries     "uint16_t"      [[Number of used occlusion queries.]]
+	.numPrograms             "uint16_t"      [[Number of used programs.]]
+	.numShaders              "uint16_t"      [[Number of used shaders.]]
+	.numTextures             "uint16_t"      [[Number of used textures.]]
+	.numUniforms             "uint16_t"      [[Number of used uniforms.]]
+	.numVertexBuffers        "uint16_t"      [[Number of used vertex buffers.]]
+	.numVertexDecls          "uint16_t"      [[Number of used vertex declarations.]]
+	.textureMemoryUsed       "int64_t"       [[Estimate of texture memory used.]]
+	.rtMemoryUsed            "int64_t"       [[Estimate of render target memory used.]]
+	.transientVbUsed         "int32_t"       [[Amount of transient vertex buffer used.]]
+	.transientIbUsed         "int32_t"       [[Amount of transient index buffer used.]]
+--	.numPrims                "uint32_t[Topology::Count]"      [[Number of primitives rendered.]]
```

Comment:
Count is enum.

Reply:
I see.

Reply:
https://github.com/cloudwu/bgfxidl/commit/8f5353309e98a6433171665df4e7a585f2a2b70c

ok now